app/views/admin/orders/index.html.erb

```diff
@@ -7,6 +7,7 @@
       <th>Description</th>
       <th>Price</th>
       <th>Deliver method</th>
+      <th>Reward code</th>
```

Comment:
I am not sure, if I would put it here - displaying it on the show page, should be enough